src/App.tsx

```diff
@@ -1,27 +1,76 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos, getUsers } from './api/api';
+
+const defaultUser = {
+  id: 0,
+  createdAt: '',
+  updatedAt: '',
+  name: '',
+  username: '',
+  email: '',
+  phone: '',
+  website: '',
+};
 
 const App: React.FC = () => {
   const [selectedUserId, setSelectedUserId] = useState(0);
+  const [todosList, setTodosList] = useState<Todo[]>([]);
+  const [currentUser, setCurrentUser] = useState<User>(defaultUser);
```

Comment:
maybe move this state to User component and fetch it there. You only need to pass userId there